project/AppDependencies.scala

```diff
@@ -7,9 +7,9 @@ object AppDependencies {
   val compile = Seq(
     ws,
     "uk.gov.hmrc"       %% "domain"                     % "6.2.0-play-28",
-    "uk.gov.hmrc"       %% "mongo-lock"                 % "7.0.0-play-28",
+    "uk.gov.hmrc"       %% "mongo-lock"                 % "7.1.0-play-28",
```

Comment:
I think new hmrc mongo lib has locking mechanism and we don't need a separate lib for it. what do you thnk?